tasks/get_localization.js

```diff
@@ -25,24 +25,18 @@ module.exports = function(grunt) {
       }
     }
 
-    var ignorePatterns = [/^\./, /^en_US$/, /^compendia$/, /^templates$/];
-    var locales = fs.readdirSync('locale')
-    .filter(function(name) {
-      var stat = fs.statSync(path.join('locale', name));
-      // is a directory, and it matches none of the ignore patterns.
-      if (!stat.isDirectory()) {
-        return false;
-      }
-      for (var i = 0; i < ignorePatterns.length; i++) {
-        if (ignorePatterns[i].exec(name)) {
-          return false;
-        }
-      }
-      return true;
-    })
-    .map(function(name) {
-      return name.replace('_', '-');
-    });
+    var locales = [
```

Comment:
Yeah, which isn't great. Do you know another place we could get this list? Maybe we need to make an API in Sumo for it?